package.json

```diff
@@ -217,7 +218,7 @@
     "pack:tarballs": "oclif pack:tarballs",
     "pack:verify": "sf-release cli:tarballs:verify",
     "pack:win": "oclif pack:win --additional-cli sf",
-    "postinstall": "npm install @salesforce/cli -g",
+    "postinstall": "sfdx whatsnew --hook && npm install @salesforce/cli -g",
```

Comment:
Might want to do `npm install @salesforce/cli -g` first so that release notes appear at the bottom of the output where users are more likely to see them.

Reply:
Also, we 'bout to merge this so you can probs just take it off now :)
https://github.com/salesforcecli/sfdx-cli/pull/351